applications/nrf_desktop/description.rst

```diff
@@ -1044,9 +1044,33 @@ See the following list of possible scenarios and best practices:
 * Radio frequency (RF) noise can negatively affect the HID report rate for wireless connections.
   If a HID report fails to be delivered in a given Bluetooth LE LLPM connection event, it is retransmitted in the subsequent connection event, which effectively reduces the report rate.
   By avoiding congested RF channels, the :ref:`nrf_desktop_ble_qos` helps to achieve better connection quality and a higher report rate.
-* For the USB device connected directly, you can configure your preferred USB HID poll interval using the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option.
-  By default, the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option is set to ``1`` to request the lowest possible poll interval.
-  Set parameters are not enforced, meaning that the HID host may still eventually use a value greater than the USB polling interval requested by a peripheral.
+* For the USB device connected directly, depending on used USB stack:
+
+  * If you use USB legacy stack, you can configure your preferred USB HID poll interval using the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option.
+    By default, the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option is set to ``1`` to request the lowest possible poll interval.
+  * If you use USB next stack, you can configure your preffered USB HID polling rate using the ``in-polling-rate`` property of DTS node compatible with ``zephyr,hid-device``.
+    The lowest polling rate that is supported by the USB High-Speed is ``125`` us which corresponds to ``8`` kHz report rate.
```

Comment:
```suggestion
    The lowest polling rate that is supported by the USB High-Speed is 125 us, which corresponds to an 8 kHz report rate.
```
tbc - are these values set in a Kconfig option? If not, there's no need to present it as an input value.

Reply:
They are set in DTS